AppleARM.md

```diff
@@ -0,0 +1,178 @@
+# Oracle Storage Adapter for Parse Server on Apple Silicon
+
+This document describes how to build and run [Parse Server](https://parseplatform.org/) with the new Oracle storage adapter on [Apple Silicon](https://en.wikipedia.org/wiki/Apple_silicon). It will demonstrate running against the [Free Oracle Database 23ai Docker container](https://www.oracle.com/database/free) using the [Oracle NodeJS libraries](https://node-oracledb.readthedocs.io/en/latest).
+
+## Prerequisites
+
+The Oracle SQL client is a software application that allows users to connect to an Oracle database and execute queries and manage the database.
+
+[SQL Client](https://www.oracle.com/database/sqldeveloper/technologies/sqlcl/download/)
+
+The Oracle Instant Client is a set of software libraries that allow users to connect to an Oracle database without a full Oracle database installation.
+
+[Instant Client Libraries](https://www.oracle.com/cis/database/technologies/instant-client/downloads.html)
+
+## Installation
+
+Clone [Parse Server Repository](https://github.com/parse-community/parse-server). Supported version 7.3.0 and above.  
+
+There were changes in this release that supported:
+
+1. [Test Exclusion List](https://github.com/parse-community/parse-server/pull/8774)
+2. [Dynamic Database Adapter configuration](https://github.com/parse-community/parse-server/pull/8883)
+
+```
+git clone --depth 1 --branch 7.3.0 https://github.com/parse-community/parse-server.git
+cd parse-server
+```
+2. Clone this Oracle Samples repo into src/Adapters/Storage/Oracle
+ ```
+cd src/Adapters/Storage
+git clone https://github.com/oracle-samples/oracleadapter-parse.git Oracle
+cd Oracle
+rm -rf .git    # IMPORTANT or build will fail
+cd ../../../.. # Go back to Project Root
+```
+
+## Getting Started
+### Building Parse with Oracle Storage Adapter
+1. Add the Oracle database dependency
+
+    ```
+    npm install oracledb@6.6.0
+    ```
+
+    [Quick Start node-oracledb Installation](https://node-oracledb.readthedocs.io/en/latest/user_guide/installation.html#quick-start-node-oracledb-installation)
+
+2. Add the Parse File Adapter dependency
+
+    ```
+    npm install --save @parse/fs-files-adapter
+    ```
+
+    This defaults to local storage. 
+
+    [Parse Server File Storage Adapter Repository](https://github.com/parse-community/parse-server-fs-adapter)
+
+3. Run 
+    ```
+    npm ci
+    ``` 
+    to build the server
+
+## How To Run
+### Configuring Free23ai Oracle database image
+1. Get and Start the image
+
+    ```
+    docker run --name free23ai -d -p 1521:1521 -e ORACLE_PASSWORD=Welcome12345 -e APP_USER=testuser -e APP_USER_PASSWORD=Welcome12345 gvenzl/oracle-free:23.5-slim-faststart
```

Comment:
Don't know if it matters but there is an official image ` container-registry.oracle.com/database/free:23.5.0.0-lite-arm64`